### DISCUSSION TITLE
can add Pagination Component ?

### DISCUSSION THREAD

**Discussion by jiulu313** [chakra-ui/chakra-ui#8734] [Ideas]
### Description

want a  Pagination Component 

### Problem Statement/Justification

 Pagination Component 

### Proposed Solution or API

 Pagination Component 

### Alternatives

 Pagination Component 

### Additional Information

 Pagination Component

**Comment by segunadebayo** [comment]
`Pagination` shipped. https://chakra-ui.com/docs/components/pagination
